apps/site/lib/site_web/templates/event/_event_list.html.eex

```diff
@@ -1,13 +1,21 @@
 <div class="page-section event-listing">
-  <%= content_tag(:h2, @title, extra_classes) %>
   <!-- This list-group stuff is bootstrap -->
   <ul class="list-group list-group-flush">
+    <!-- Previous months button. A JS script sets up a listener for button click -->
+    <li class="list-group-item m-previous-events-button">
+      <a class="m-previous-events-button">
+        View Previous <%= pretty_date(Timex.now("America/New_York"), "{Mfull} {YYYY}") %> Events
+        <i class="fa fa-angle-down down" aria-hidden="true"></i>
+      </a>
+    </li>
     <%= for event_teaser <- @events do %>
       <%
         range = %{start: event_teaser.date, stop: event_teaser.date_end}
         path_fn = fn -> cms_static_page_path(@conn, event_teaser.path) end
       %>
-      <li class="list-group-item u-flex-container m-event">
+      <!-- If date is upcoming, display on init -->
+      <!-- Otherwise, add a 'hidden' class. On previous-button click, this class is removed -->
+      <li class="<%= if date_upcoming(range), do: 'list-group-item u-flex-container m-event', else: 'list-group-item u-flex-container m-event m-hidden-event' %>">
```

Comment:
Since the else is optional I think you could also do `class="list-group-item u-flex-container m-event <%= if date_upcoming(range), do: 'm-hidden-event' %>"`